README.md

```diff
@@ -332,6 +333,14 @@ A curated list of awesome Java frameworks, libraries and software.
 * [OpenTSDB](http://opentsdb.net) - Scalable and distributed time series database written on top of Apache HBase.
 * [OrientDB](https://orientdb.com/orientdb/) - Embeddable distributed database written on top of Hazelcast.
 
+## Distributed Transactions
+
+*Distributed transactions provide a mechanism for ensuring consistency of data updates in the presence of concurrent access and partial failures.*
+
+* [Atomikos](https://www.atomikos.com) - Provides transactions for REST, SOA and microservices, also with support for JTA and XA.
```

Comment:
There's a company behind Atomikos but I believe there's a "free" download of the source (no free repo afaik). If the distinction is important let me know how best to proceed.